appveyor.yml

```diff
@@ -0,0 +1,31 @@
+environment:
+  matrix:
+
+    - PYTHON: "C:\\Python35"
+      PYTHON_VERSION: "3.5.1"
+      PYTHON_ARCH: "32"
+      
+    - PYTHON: "C:\\Python35-x64"
+      PYTHON_VERSION: "3.5.1"
+      PYTHON_ARCH: "64"
+    
+    - PYTHON: "C:\\Python36"
+      PYTHON_VERSION: "3.6.x"
+      PYTHON_ARCH: "32"
+      
+    - PYTHON: "C:\\Python36-x64"
+      PYTHON_VERSION: "3.6.x"
+      PYTHON_ARCH: "64"
+     
+     
+install:
+   - "%PYTHON%\\python.exe -m pip install -r test-requirements.txt"
+   - "git submodule update --init typeshed"
+   - "%PYTHON%\\python.exe setup.py install"
+   
+build: off
+
+test_script:
+  # we ignore test-cmdline, due to path differences and other tests that need lxml. extensions and check don't have tests that are discovered
+  - "%PYTHON%\\python.exe runtests.py -x lint -x check -x extensions -x reports -x cmdline"
```

Comment:
I think I have a fix for the check extensions: https://github.com/python/mypy/commit/540974645164189c0733af8db0b3c19f2f57cbfe

I also understand why the cmdline test is failing: there's a test in test-data/unit/cmdline.test, named testCoberturaParser, that tests the --cobertura-xml-report flag, and that flag is being ignored because lxml isn't available. But I don't think there is a way to skip just one testcase (though I'll try one more thing).